.gitignore

```diff
@@ -6,3 +6,10 @@ harborcreds.yaml
 mavencreds.yaml
 temp.zip
 galasabuild-image/galasabld-linux-amd64
+.DS_STORE
+
+openapi2beans/build
+
+openapi2beans/JavaChecker/.vscode
```

Comment:
I would change this to **/.vscode as we shouldn't ever need to check-in IDE config files.

Reply:
corrected and added